pkgs/tools/networking/networkmanager/fix-paths.patch

```diff
@@ -78,22 +78,12 @@ index bebc53a851..93710455d5 100644
      g_ptr_array_add(argv, (char *) arg1);
  
 diff --git a/src/libnmc-base/nm-vpn-helpers.c b/src/libnmc-base/nm-vpn-helpers.c
-index e2c0c394bd..1a507aa0d4 100644
+index 7ad5bee509..f82063afa8 100644
 --- a/src/libnmc-base/nm-vpn-helpers.c
 +++ b/src/libnmc-base/nm-vpn-helpers.c
-@@ -198,25 +198,8 @@ nm_vpn_openconnect_authenticate_helper(const char *host,
-     gs_free const char **output_v = NULL;
-     const char *const   *iter;
-     const char          *path;
--    const char *const    DEFAULT_PATHS[] = {
```

Comment:
Should not we still remove this variable? It would make it easier to detect if another use of it is added.

Reply:
Thank you, good catch. This was an accident while I was cleaning up the patches, not an intentional change :-)